platform-sdk/swirlds-common/src/main/java/com/swirlds/common/metrics/platform/DefaultIntegerPairAccumulator.java

```diff
@@ -31,7 +31,7 @@
 /**
  * Platform-implementation of {@link IntegerPairAccumulator}
  */
-public class DefaultIntegerPairAccumulator<T> extends DefaultMetric implements IntegerPairAccumulator<T> {
+public class DefaultIntegerPairAccumulator<T> extends AbstractPlatformMetric implements IntegerPairAccumulator<T> {
```

Comment:
The chosen names are not consistent.
For one instance you renamed DefaultMetric was separated intoAbstractMetric and DefaultPlatformMetric.
We have DefaultCounter, DefaultGauge, etc all extending AbstractMetric in the metrics-impl module. 
On the other hand, although we have: PlatformIntegerAccumulator, PlatformDoubleAccumulator on the commons module, we also have DefaultDurationGauge, DefaultIntegerPairAccumulator, DefaultFunctionGauge extending AbstractPlatformMetric. 


So my proposal:
 a) remove AbstractPlatformMetric, Platform metrics can extend AbstractMetric or some other DefaultMetric. As mentioned above, maybe a default method implementation in the interface is enough. 
 b) reincorporate "default" to the name AbstractMetric as AbstractDefaultMetric, or if we still want to remove it we need to remove it from the rest of the classes too, to make it consistent.
 c) All classes extending PlatformMetric interface should be called "Platform...Metric" and remove the default of the name.